git_machete/cmd.py

```diff
@@ -1902,7 +1917,8 @@ def parse_direction(b, down_pick_mode):
         elif cmd == "file":
             expect_no_param(parse_options(args))
             # No need to read definition file.
-            print(definition_file)
+            abs_definition_file = os.path.abspath(os.path.join(get_abs_git_dir(), os.pardir, definition_file))
```

Comment:
Yeah, you call `abspath` here... I can't see a reason why we just can't use `abspath` in `get_abs_git_dir` instead

Reply:
Also, I'd rather store the `definition_file` path as absolute from the very beginning... so line 1866 would look like: `definition_file = os.path.join(get_abs_git_dir(), "machete")`